docs/proposals/Release-Supply-Chain-Metadata.md

```diff
@@ -0,0 +1,178 @@
+# Supply Chain Metadata for Ratify Assets
+
+Ratify currently publishes multiple forms of release assets both for production and development uses. Currently, these assets are not published with accompanying supply chain metadata including signatures, SBOMs, and provenance. Shipping each of these forms of metadata with all binaries and container images produced by Ratify will provide consumers a verifiable way to guarantee integrity of Ratify assets. Furthermore, this will improve Ratify's OSSF scorecard.
```

Comment:
In terms of OSSF scorecard, we may add a reference link to https://github.com/ossf/scorecard/blob/main/docs/checks.md#binary-artifacts and mention that the goal is to fix the major risks marked as High and Medium.